### PR TITLE
Provide Reference guide for the Kanto Local Digital Twins

### PR DESCRIPTION
**Pull request by antonia-avramova**
[#47] Provide Reference guide for the Kanto Local Digital Twins

Signed-off-by: Antonia Avramova <antonia.avramova@bosch.io>

### CODE REVIEW

**Comment by e-grigorov** [comment]
@antonia-avramova The Eclipse Kanto project is currently restructuring its license scheme - from Eclipse Public License 2.0 only to dual licensing under Eclipse Public License 2.0 **and** Apache License 2.0 (you can read more about it [here](https://projects.eclipse.org/projects/iot.kanto/reviews/restructuring-review)).

This contribution will be merged into the project with this new licensing scheme so its important for the project and the community to have your agreement on proceeding with it given these new circumstances :blush: 

Please, provide your feedback as a comment here that you, as an author of this contribution, understand and agree with the change explained above. Thank you!

**Comment by antonia-avramova** [comment]
> @antonia-avramova The Eclipse Kanto project is currently restructuring its license scheme - from Eclipse Public License 2.0 only to dual licensing under Eclipse Public License 2.0 **and** Apache License 2.0 (you can read more about it [here](https://projects.eclipse.org/projects/iot.kanto/reviews/restructuring-review)).
> 
> This contribution will be merged into the project with this new licensing scheme so its important for the project and the community to have your agreement on proceeding with it given these new circumstances 😊
> 
> Please, provide your feedback as a comment here that you, as an author of this contribution, understand and agree with the change explained above. Thank you!

I understand and agree with the license change explained above.

**Review by e-grigorov — APPROVED** [review]

File: web/site/content/docs/references/local-digital-twins-config.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+| **Logging** | | | |
+| logFile | string | log/local-digital-twins.log | Path to the file where log messages are written |
+| logLevel | string | INFO | All log messages at this or higher level will be logged, the log levels in descending order are: ERROR, WARN, INFO, DEBUG and TRACE |
+| logFileCount | int | 5 | Log file maximum rotations count |
+| logFileMaxAge | int | 28 | Log file rotations maximum age in days, use 0 to not remove old log files |
+| logFileSize | int | 2 | Log file size in MB before it gets rotated |
+
+### Example
+
+The minimal required configuration to enable the local digital twins and their synchronization with publicly available {{% refn "https://www.eclipse.org/hono/sandbox/" %}}Eclipse Hono sandbox{{% /refn %}}.
```
Comment (konstantina-gramatova) [inline web/site/content/docs/references/local-digital-twins-config.md:52]: ... the local digital twins and their synchronization with publicly available {{% refn ....
should be 

... the local digital twins and their synchronization with the publicly available {{% refn ....

File: web/site/content/docs/references/local-digital-twins-config.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+| localAddress | string | tcp://localhost:1883 | Address of the MQTT server/broker that the local digital twins will connect for the local communication, the format is: `scheme://host:port` |
+| localUsername | string | | Username that is a part of the credentials |
+| localPassword | string | | Password that is a part of the credentials |
+| **Local connectivity - TLS** | | | |
+| localCACert | string | | PEM encoded CA certificates file |
+| localCert | string | | PEM encoded certificate file to authenticate to the MQTT server/broker |
+| localKey | string | | PEM encoded unencrypted private key file to authenticate to the MQTT server/broker |
+| **Logging** | | | |
+| logFile | string | log/local-digital-twins.log | Path to the file where log messages are written |
+| logLevel | string | INFO | All log messages at this or higher level will be logged, the log levels in descending order are: ERROR, WARN, INFO, DEBUG and TRACE |
```
Comment (konstantina-gramatova) [inline web/site/content/docs/references/local-digital-twins-config.md:45]: All log messages at this or higher level will be logged, ....
 
 should be 
 
  All log messages at this or a higher level will be logged, ....

**Review by konstantina-gramatova — CHANGES_REQUESTED** [review]

**Review by konstantina-gramatova — APPROVED** [review]

**Pull request merged**